### PR TITLE
3.0 preview dev

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

File: paper-dropdown-menu-light.js [hunk 1]
```diff
@@ -54,19 +75,27 @@ The following custom properties and mixins are also available for styling:
...
+`--paper-dropdown-menu-icon` | A mixin that is applied to the internal icon |
+`{}`
+`--paper-dropdown-menu-disabled-opacity` | The opacity of the dropdown when
+disabled  | `0.33`
+`--paper-dropdown-menu-color` | The color of the input/label/underline when the
+dropdown is unfocused | `--primary-text-color`
+`--paper-dropdown-menu-focus-color` | The color of the label/underline when the
+dropdown is focused  | `--primary-color`
+`--paper-dropdown-error-color` | The color of the label/underline when the
+dropdown is invalid  | `--error-color`
```
Comment (bicknellr) [inline paper-dropdown-menu-light.js]: This table was broken by the formatter. ([fix here](https://github.com/PolymerElements/paper-item/pull/125#discussion_r202196878))

File: paper-dropdown-menu.js [hunk 2]
```diff
@@ -48,11 +70,16 @@ The following custom properties and mixins are also available for styling:
...
+`--paper-dropdown-menu-disabled` | A mixin that is applied to the element host
+when disabled | `{}`
+`--paper-dropdown-menu-ripple` | A mixin that is applied to the internal ripple
+| `{}`
+`--paper-dropdown-menu-button` | A mixin that is applied to the internal menu
+button | `{}`
+`--paper-dropdown-menu-input` | A mixin that is applied to the internal paper
+input | `{}`
+`--paper-dropdown-menu-icon` | A mixin that is applied to the internal icon |
+`{}`
```
Comment (bicknellr) [inline paper-dropdown-menu.js]: Also broken by the formatter.

**Comment by bicknellr** [comment]
I guess the `package-lock.json` thing is just going to be continuous as other packages get updated. So as long as that was regenerated after making changes to the `package.json`, I bet it's ok.

File: paper-dropdown-menu-light.js [hunk 3]
```diff
@@ -80,28 +109,6 @@ To style it:
 @hero hero.svg
```
Comment (bicknellr) [inline paper-dropdown-menu-light.js]: This line can be removed. There's also one [in `paper-dropdown-menu.js`](https://github.com/PolymerElements/paper-dropdown-menu/pull/298/files#diff-2951e6f740f646de1ce1b7f94aa0823dR90).

**Comment by keanulee** [comment]
@bicknellr Demo style works correctly for me - is this what you see?
image

**Comment by bicknellr** [comment]
Specifically the last one (with the label "You can style a paper-dropdown-menu using custom properties") doesn't seem to be working. All the others look ok though.

**Comment by bicknellr** [comment]
Also, could you merge master into this branch? There's a PR in there which switches the code owner.

**Pull request merged**